pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -177,6 +177,11 @@ with generated;
     dependencies = ["nvim-yarp"];
   });
 
+  ncm2-jedi = ncm2-jedi.overrideAttrs(old: {
+    dependencies = ["nvim-yarp"];
```

Comment:
Is this not covered by `ncm2` already or is there a way to use ncm2-jedi without?

Reply:
It would be covered if `ncm` was a dependency. I wasn't entirely sure if it should be, since I guess the ncm2-jedi provider could in theory be used by another completion plugin.

Also nvim-yarp is used directly, so its nicer to have it as an explicit dependency anyways.

Reply:
It uses also ncm2's api: https://github.com/ncm2/ncm2-jedi/blob/0418d5ca8d4fe6996500eb04517a946f7de83d34/autoload/ncm2_jedi.vim#L37

Reply:
Thats convincing. I added a dependency to ncm2 (still keeping the nvim-yarp dependency because it is also a direct dependency, although it does not matter in practice).